scss/_base_grid-definitions.scss

```diff
@@ -51,4 +51,31 @@
       }
     }
   }
+
+  // Grid
+  %vf-grid-container-padding {
```

Comment:
Could these be inside vf-b-grid-definitions mixin? So they are not included multiple times by mistake?

Reply:
well spotted, thanks. fixed along with a typo in the md file.